admin-dev/themes/new-theme/js/pages/product/product-map.ts

```diff
@@ -260,4 +260,7 @@ export default {
       deleteBtn: '.js-delete-specific-price-btn',
     },
   },
+  packedProducts: {
+    searchInput: '#product_description_packed_products',
```

Comment:
Considering the ID here, I guess you didn't move the Packs in the stock tab yet?

Reply:
moved in stock